components/infobox/wikis/starcraft2/infobox_person_custom_shared.lua

```diff
@@ -192,7 +192,7 @@ function CustomPerson.getPersonType()
 	role = string.lower(role or '')
 	local category = _ROLES[role]
 	local store = category or _CLEAN_OTHER_ROLES[role] or _args.defaultPersonType
-	if category == 'Map Maker' then
+	if category == 'Map maker' then
```

Comment:
```suggestion
	if category == _ROLES['map maker'] then
```
Could do it like this to make it less hardcoded. However why is it needed?

Reply:
> However why is it needed (why not update the data table)?

because the role stored in lpdb should be `Map maker`